modules/sdk/src/main/scala/org/tessellation/sdk/domain/seedlist.scala

```diff
@@ -0,0 +1,77 @@
+package org.tessellation.sdk.domain
+
+import cats.Order
+import cats.syntax.contravariant._
+import cats.syntax.either._
+import cats.syntax.option._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import com.comcast.ip4s.IpAddress
+import derevo.cats.eqv
+import derevo.derive
+import fs2.data.csv._
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import io.estatico.newtype.macros.newtype
+
+object seedlist {
+
+  @derive(eqv)
+  @newtype
+  case class Alias(value: String)
+
+  object Alias {
+
+    implicit val maybeAliasCellDecoder: CellDecoder[Option[Alias]] = CellDecoder.stringDecoder.map { stringValue =>
+      if ("".equals(stringValue)) none else Alias(stringValue).some
+    }
+
+  }
+
+  case class SeedlistEntry(
+    peerId: PeerId,
+    ipAddress: Option[IpAddress],
+    alias: Option[Alias],
+    bias: Option[TrustValueRefined]
+  )
+
+  object SeedlistEntry {
+
+    implicit val maybeIpAddressCellDecoder: CellDecoder[Option[IpAddress]] =
+      CellDecoder.stringDecoder.emap { value =>
+        if ("".equals(value))
+          none.asRight[DecoderError]
+        else
+          IpAddress.fromString(value) match {
+            case v @ Some(_) => v.asRight[DecoderError]
+            case None        => Left(new DecoderError("Invalid IP address."))
```

Comment:
Please add a value there as well if that's possible. Finding an invalid entry within hundreds may be hard

Reply:
Added the invalid value to the error's message.